### PR TITLE
Add trend service to organize trend parser functions

### PR DESCRIPTION
**Pull request by alexcn3**

### CODE REVIEW

File: project/src/main/java/com/google/sps/data/TrendService.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.net.URL;
+import java.util.List;
+import com.google.sps.data.Trend;
+import com.google.sps.data.TrendRSS;
+import com.google.sps.data.TrendRSSFeed;
+import com.google.sps.data.TrendFrequencyParser;
+
```
Comment (niuamy) [inline project/src/main/java/com/google/sps/data/TrendService.java:35]: nit: maybe add comment here

File: project/src/main/java/com/google/sps/servlets/TrendServlet.java [hunk 2]
```diff
@@ -14,25 +14,13 @@
...
 import javax.servlet.http.HttpServletResponse;
-import java.util.ArrayList;
-import java.net.URL;
-import java.util.List;
-import com.google.gson.Gson;
-import com.google.sps.data.Trend;
-import com.google.sps.data.TrendRSS;
-import com.google.sps.data.TrendRSSFeed;
-import com.google.sps.data.TrendFrequencyParser;
+import com.google.sps.data.TrendService;
```
Comment (niuamy) [inline project/src/main/java/com/google/sps/servlets/TrendServlet.java:24]: nit: maybe add comment here as well

File: project/src/main/java/com/google/sps/data/TrendService.java (hunk 1 above)
Comment (ashley3) [reply project/src/main/java/com/google/sps/data/TrendService.java:35]: Agreed, comment Javadoc please :)

File: project/src/main/java/com/google/sps/servlets/TrendServlet.java (hunk 2 above)
Comment (ashley3) [reply project/src/main/java/com/google/sps/servlets/TrendServlet.java:24]: +1

**Pull request merged**